metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/StringFunctions.java

```diff
@@ -222,4 +222,16 @@ public Object apply(List<Object> args) {
       return null;
     }
   }
+
+  @Stellar( name="LENGTH"
+          , description = "Returns the length of a string"
+          , params = { "input - String" }
+          , returns = "String"
```

Comment:
Probably need to change`returns = "String"` to indicate that it returns an Integer which is the length of the String.  However you want to say that is fine, but it does not actually return a String.